File: TestPlatform/src/Services/TestManager/VietGeeks.TestPlatform.TestManager.WebApp/ClientApp/apps/test-runner/src/app/api/models/activate-question-output.ts
/* tslint:disable */
/* eslint-disable */
import { ExamQuestion } from './exam-question';
export interface ActivateQuestionOutput {
  activationResult?: boolean;
  activeQuestion?: ExamQuestion;
  activeQuestionId?: null | string;
  activeQuestionIndex?: null | number;
  answerIds?: null | Array<string>;
  canFinish?: boolean;
}

File: TestPlatform/src/Services/TestRunner/VietGeeks.TestPlatform.TestRunner.Contract/ProctorExamActor/ActivateQuestionOutput.cs
namespace VietGeeks.TestPlatform.TestRunner.Contract.ProctorExamActor;

public class ActivateQuestionOutput: IActiveQuestion
{
    public bool ActivationResult { get; set; }
    public string? ActiveQuestionId { get; set; }
    public int? ActiveQuestionIndex { get; set; }
    public ExamQuestion? ActiveQuestion { get; set; }
    public bool CanFinish {get;set;}
    public string[]? AnswerIds { get; set; } = default!;
}